src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java

```diff
@@ -58,7 +68,7 @@ public class Reference extends InformationContentEntity {
 	 * Retrieve PMID if available in the crossReference collection otherwise MOD ID
 	 */
 	@Transient
-	@JsonIgnore
+	@JsonView(View.ForPublic.class)
```

Comment:
Does this need the view?

Reply:
Makes life a lot easier - we have it for pubModID, so don't know why we don't have this too